src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { bytes_to_hex, Sha256 } from "asmcrypto.js"
+import { TFileInformation, TdrFile } from "./types"
+
+export const getFileInformation: TFileInformation = async (
```

Comment:
I wonder if we should call it `fileMetadata` rather than `fileInformation`? David C might have an opinion on this, especially if we're releasing this a general-purpose library.

I also wonder if there's a better word than `get`, which sounds quite Java-y? `extract`? `calculate`?

Reply:
Yes, that's much better :smile: I was having real trouble naming things. I've called the function `extractFileMetadata` and  I've renamed the repo `tdr-file-metadata` as well